azext_iot/tests/iothub/message_endpoint/test_iothub_message_endpoint_int.py

```diff
@@ -143,6 +143,83 @@ def test_iot_eventhub_endpoint_lifecycle(provisioned_event_hub_with_identity_mod
     assert len(eventhub_list) == 3
     assert endpoint_list["eventHubs"] == eventhub_list
 
+    # Update
+    # Keybased -> System
```

Comment:
this is somewhat minimal testing compared to what the user can do with the endpoints (specifically I skip changing to a different resource (could be shared access policy key for keybased, eventhub, eventhub namespace; full testing will require having a second eventhub namespace and making more eventhubs and policies within one namespace)

Reply:
Will say this is ok for now since unit testing exists for this command